oracle-price-monitor/package.json

```diff
@@ -21,7 +21,11 @@
     "test": "jest"
   },
   "dependencies": {
-    "forta-agent": "^0.0.38"
+    "forta-agent": "^0.0.38",
+    "axios": "^0.26.1",
+    "bignumber.js": "^9.0.2",
+    "dotenv": "^16.0.0",
+    "ethers": "^5.6.4"
```

Comment:
```suggestion
    "forta-agent": "^0.0.38"
```

Reply:
cleaned up all packages in commit 053c6dd456d5d84c7965fe4c064d116817490f5d